src/object/endsequence_fireworks.cpp

```diff
@@ -40,13 +41,22 @@ void
 EndSequenceFireworks::starting()
 {
   EndSequence::starting();
-  endsequence_timer.start(7.3f * ScreenManager::current()->get_speed());
-  Sector::current()->add_object(std::make_shared<Fireworks>());
+  // Delay fireworks a bit, to see the boss falling off the screen
```

Comment:
I'm not happy with this solution. Fireworks might be used elsewhere too. What about a separate sequence for the bosses that will trigger the fireworks sequence automatically?